### DISCUSSION TITLE
update ALOS annual mosaics

### DISCUSSION THREAD

**Discussion by KBodolai** [microsoft/PlanetaryComputer#358] [Data request]
JAXA recently released the 2023 version of the ALOS annual mosaics. The collection currently available in the planetary computer (https://planetarycomputer.microsoft.com/dataset/alos-palsar-mosaic) only includes data up to 2021, would it be possible to run an update? 

Thanks!

**Comment by TomAugspurger** [comment]
Thanks for the request. We'll take a look when we get a chance.

**Reply by KBodolai** [reply]
thank you Tom!

**Comment by KBodolai** [comment]
Hello, just to add to this - the 2024 mosaic is now released as well. Are there any plans to update these collections?

Thanks!
